assets/js/atomic/components/product/button/index.js

```diff
@@ -3,129 +3,162 @@
  */
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
-import apiFetch from '@wordpress/api-fetch';
 import { __, sprintf } from '@wordpress/i18n';
-import { Component } from 'react';
-import { addQueryArgs } from '@wordpress/url';
+import {
+	useMemo,
+	useCallback,
+	useState,
+	useEffect,
+	useRef,
+} from '@wordpress/element';
+import { useDispatch } from '@wordpress/data';
+import { find } from 'lodash';
```

Comment:
Are we reverting the decision to avoid lodash now we're using more WP libraries on frontend?

Reply:
`lodash` is a dependency for `@wordpress/element` and `@wordpress/data` and `@wordpress/data-controls` so it's unavoidable currently for on the frontend.  We'll need to discuss this, but I think we should use these dependencies and explore other options for controlling bundle size on the frontend.  One option will be to do tree-shaked builds that load dependencies directly vs package imports (vs the wp built bundles).  It is something we'll have to look into but I don't think we avoid lodash in the short term.